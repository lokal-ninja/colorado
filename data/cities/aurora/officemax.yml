id: 3540423351
type: node
latitude: 39.69512
longitude: -104.82405
postcode: "80012"
city: Aurora
street: "East Mississippi Avenue"
housenumber: 14030
phone: "+1-303-752-0040"
opening_hours: "Su-Mo 10:00-18:00, Tu-Sa 08:00-21:00"
website: "https://www.officedepot.com/storelocator/co/aurora/officemax-6293/?cm_mmc"
