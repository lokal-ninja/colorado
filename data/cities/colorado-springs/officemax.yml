id: 1282330343
type: way
latitude: 38.79331
longitude: -104.80206
postcode: "80906"
city: Colorado Springs
street: "East Cheyenne Mountain Boulevard"
housenumber: 1640
phone: "+1-719-527-6560"
opening_hours: "Mo-Fr 09:00-20:00; Sa 10:00-19:00; Su 11:00-18:00"
website: "https://www.officedepot.com/storelocator/co/colorado-springs/office-depot-6554"