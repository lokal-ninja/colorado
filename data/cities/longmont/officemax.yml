id: 514868782
type: way
latitude: 40.15344
longitude: -105.1118
postcode: "80501"
city: Longmont
street: "Ken Pratt Boulevard"
housenumber: 1120
phone: ""
opening_hours: ""
website: "https://www.officedepot.com/storelocator/co/longmont/officemax-6251"