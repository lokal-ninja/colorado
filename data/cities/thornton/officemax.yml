id: 45610957
type: way
shop: office supplies
latitude: 39.9167764
longitude: -104.97630620000001
postcode: "80233"
city: Thornton
street: "East 120th Avenue"
housenumber: 801
phone: "+1-303-920-3486"
opening_hours: ""
website: "https://www.officedepot.com/storelocator/co/thornton/officemax-6708"
