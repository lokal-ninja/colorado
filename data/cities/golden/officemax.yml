id: 4006328679
type: node
shop: office supplies
latitude: 39.7356313
longitude: -105.162462
postcode: "80401"
city: Golden
street: "West Colfax Avenue"
housenumber: 14375
phone: ""
opening_hours: ""
website: ""
